dp/cloud/python/magma/test_runner/tests/test_dp_with_orc8r.py

```diff
@@ -295,11 +321,16 @@ def then_cbsd_is(self, actual: Dict[str, Any], expected: Dict[str, Any]):
             del grant['transmit_expire_time']
         self.assertEqual(actual, expected)
 
-    def then_cbsd_is_deleted(self):
-        r = send_request_to_backend('get', 'cbsds')
+    def then_cbsd_is_deleted(self, serial_number: str):
+        params = {}
```

Comment:
there is a lot of duplication between this method and `when_cbsd_is_fetched` method, can common part be extracted?

Reply:
Added private check method.